packages/core/src/libraries/passcode.ts

```diff
@@ -91,12 +97,16 @@ export const passcodeExpiration = 10 * 60 * 1000; // 10 minutes.
 export const passcodeMaxTryCount = 10;
 
 export const verifyPasscode = async (
-  sessionId: string,
+  sessionId: string | undefined,
```

Comment:
nit: was thinking that should we align `jti` in `createPasscode` and `sessionId` in `verifyPasscode`?

Reply:
unified